mysql-test/suite/rocksdb/t/index_merge_rocksdb.test

```diff
@@ -0,0 +1,93 @@
+--source include/have_rocksdb.inc
+# Skiping this test from Valgrind execution as per Bug-14627884
+--source include/not_valgrind.inc
+# Adding big test option for this test.
+#--source include/big_test.inc
+
+#
+# index_merge_rocksdb test copied over from index_merge_ror.inc
+#
+# Triggers issue # https://github.com/facebook/mysql-5.6/issues/604
+
+--disable_warnings
+DROP TABLE IF EXISTS t0,t1;
```

Comment:
Nit: this idiom is considered obsolete, we are removing it as we encounter on Percona Server port